qa/workunits/rbd/rbd_mirror_bootstrap.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/sh -ex
+#
+# rbd_mirror_bootstrap.sh - test peer bootstrap create/import
+#
+
+RBD_MIRROR_MANUAL_PEERS=1
+RBD_MIRROR_INSTANCES=${RBD_MIRROR_INSTANCES:-1}
+. $(dirname $0)/rbd_mirror_helpers.sh
+
+setup
+
+testlog "TEST: bootstrap cluster2 from cluster1"
+# create token on cluster1 and import to cluster2
+TOKEN=${TEMPDIR}/peer-token
+TOKEN_2=${TEMPDIR}/peer-token-2
+rbd --cluster ${CLUSTER1} mirror pool peer bootstrap create ${POOL} > ${TOKEN}
+rbd --cluster ${CLUSTER1} mirror pool peer bootstrap create ${PARENT_POOL} > ${TOKEN_2}
+cmp ${TOKEN} ${TOKEN_2}
+
+rbd --cluster ${CLUSTER2} --pool ${POOL} mirror pool peer bootstrap import ${TOKEN} --direction rx-only
+rbd --cluster ${CLUSTER2} --pool ${PARENT_POOL} mirror pool peer bootstrap import ${TOKEN} --direction rx-tx
+
+start_mirrors ${CLUSTER1}
+start_mirrors ${CLUSTER2}
+
+testlog "TEST: verify rx-only direction"
+[[ "$(rbd --cluster ${CLUSTER1} --pool ${POOL} mirror pool info --format xml |
+	${XMLSTARLET} sel -t -v  '//mirror/peers/peer[1]/uuid')" == '' ]]
```

Comment:
@dillaman Some shells do not know about `[[`. See rbd_mirror_bootstrap.sh failures in [1].

(The run had also one crash but it is not related, I created #30484 for this)

[1] http://pulpito.ceph.com/trociny-2019-09-19_12:38:56-rbd-wip-mgolub-testing-distro-basic-smithi/